packages/containers/psammead-timestamp-container/src/timestampUtilities.test.js

```diff
@@ -25,6 +27,12 @@ describe('Timestamp utility functions', () => {
     });
   });
 
+  describe('unixTimestampToMoment', () => {
+    it('should be a valid moment', () => {
+      expect(unixTimestampToMoment(timestamp)).toHaveProperty('isValid');
```

Comment:
question, the test reads as `should be a valid moment` does this mean we expect that all valid "moment" have an "isValid" property ? why not `instanceof moment` instead ?

Reply:
I don't think we can rely on the object having been created with the `moment` constructor as this isn't described in Moment's public API, whereas the presence of `isValid` should be reliable.